index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body class="page__body">
+    <header class="main-header">
+        <!--А можно сделать логотип фоном?-->
+        <a class="main-header__logo-link" href="/">
+            <img class="main-header__logo-image" src="image\mobilelogo.svg" width="33,2px" height="36.6px" alt="логотип cat energy"> </a>
+        <!--не уверена в заголовке, может надо было делать H1-->
+        <p class="main-header__title">
+            <b class="main-header__first-word">Cat</b> energy
+        </p>
+        <nav class="main-header__navigation navigtion-mobile">
+            <button class="navigtion-mobile_toogle" type="button">
+              <span class="visually-hidden">Открыть меню</span>
+            </button>
+        </nav>
+        <!--Обертка ведь тут не нужна? Можно центрировать самим header-->
+    </header>
+    <main class="index-main">
+        <article>
+            <section class="index-main__choose-programm choose-programm">
+                <div class="choose-programm__wrapper">
+                    <H1 class="choose-programm__title">Функциональное питание для котов</H1>
+                    <p class="choose-programm__subtitle">Занялся собой? Займись котом!</p>
+                    <img class="choose-programm__image" src="image\catenergy.png" width="280px" height="270px" alt="Большая банка корма.">
+                    <a class="choose-programm__button button" href="/">
+                      подобрать программу
+                    </a>
+                </div>
+            </section>
+            <section class="index-main__sistems sistems">
+                <h2 class="visually-hidden">Системы питания</h2>
+                <ul class="sistems__list">
+                    <li class="sistems__item sistem">
+                        <div class="sistem__icon--slim">
```

Comment:
`sistem__icon--slim` класс модификатора, но модификатор не может использоваться самостоятельно, так как он модифицирует свойства класса блока или элемента, поэтому рядом должен быть класс того элемента или блока, который он модифицирует. В данном случае это элемент:
```suggestion
                        <div class="sistem__icon sistem__icon--slim">
```